coq/coq-compile-common.el

```diff
@@ -20,6 +20,8 @@
   ;;(defvar coq-pre-v85 nil)
   (defvar coq-confirm-external-compilation nil); defpacustom
   (defvar coq-compile-parallel-in-background nil) ; defpacustom
+  (defvar compilation-error-regexp-alist-alist nil) ; from compile.el
+  (defvar compilation-error-regexp-alist nil) ; from compile.el
```

Comment:
This is inside (eval-when (compile)... - I am not setting these variables, I am declaring them for the compiler. Otherwise the compiler complains...